docs/en/siem/siem-ui.asciidoc

```diff
@@ -16,9 +16,10 @@ collapsible, resizable, moveable, and so forth.
 The *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* bar is available
 throughout the {siem-app} for searching and filtering.
 
-NOTE: The default index patterns for {siem-soln} events are `endgame-*`,
-`auditbeat-*`, `winlogbeat-*`, `filebeat-*`, and `packetbeat-*`. You can change
-the default index patterns in {kib} -> Management -> Advanced Settings -> `siem:defaultIndex`.
+NOTE: The default index glob patterns defined for {siem-soln} events are 
+`endgame-*`, `auditbeat-*`, `winlogbeat-*`, `filebeat-*`, and `packetbeat-*`. 
+You can change the default glob patterns in {kib} -> Management -> Advanced 
+Settings -> `siem:defaultIndex`.
```

Comment:
Just a heads up that this will be changing to include `apm-*-transcation*` as part of https://github.com/elastic/kibana/issues/52297#issuecomment-562741172. As noted in that comment, there is going to be specific logic for matching `apm-*-transcation*` to the `apm-*` Kibana Index Pattern.

Reply:
Thanks @spong 

We can update the documentation anytime - so we'll add this when it's implemented.